packages/manager/src/components/TabbedPanel/TabbedPanel.spec.js

```diff
@@ -1,66 +1,89 @@
 const { navigateToStory } = require('../../../e2e/utils/storybook');
 
 describe('Tabbed Panel Suite', () => {
-    const component = 'TabbedPanel';
-    const childStories = [
-        'default',
-    ]
-    const tabbedPanel = '[data-qa-tp]';
-    const header = '[data-qa-tp-title]';
-    const copy = '[data-qa-tp-copy]';
-    const tab = '[data-qa-tab]';
-    const tabBody = '[data-qa-tab-body]';
+  const component = 'TabbedPanel';
+  const childStories = [
+    'default',
+  ]
+  const tabbedPanel = '[data-qa-tp]';
+  const header = '[data-qa-tp-title]';
+  const copy = '[data-qa-tp-copy]';
+  const tab = '[data-qa-tab]';
+  const tabBody = '[data-qa-tab-body]';
 
-    beforeAll(() => {
-        navigateToStory(component, childStories[0]);
-    });
+  let tabValue ='';
```

Comment:
declared variable, but never used

Reply:
oof, a missed delete from a change in direction. Thanks for catching this. Fix has been pushed.

Reply:
👍